src/plugins/loading/Cargo.toml

```diff
@@ -5,6 +5,9 @@ edition.workspace = true
 
 [dependencies]
 bevy.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+uuid.workspace = true
 common.workspace = true
```

Comment:
remove dependencies from common, if possible

Reply:
Can't remove any: serde is used for some types that are (de)serializable and uuid is used in `test_tools`.